### ISSUE TITLE
Update links to GitHub

### ISSUE THREAD

**Issue by dkulp2** [openbabel/openbabel#2479]
The openbabel website still has links to sourceforge on the [wiki](http://openbabel.org/wiki/Main_Page) and in some of the sub-pages such as this [build instructions page](http://openbabel.org/wiki/Install_(source_code)). Please update these links to the GitHub repo where the latest download is available.

**Comment by nbehrnd** [comment]
This wouldn't be (so much) a problem _if_ one could create a user account on openbabel.org.  Perhaps by intent, perhaps by accident, this is not possible (anymore?).  Speaking for Firefox as provided by Linux Debian 12/bookworm (branch testing), i.e. 91.10.0esr (64-bit), the captcha function is not accepted/dysfunctional (note, the pad lock equally is crossed):

![no_admission](https://user-images.githubusercontent.com/19709084/172888477-03db8b1c-860b-4f40-96fb-aa64bb27dbf8.png)

Visiting the main page's [revision history](http://openbabel.org/w/index.php?title=Main_Page&action=history) suggests only the project's elders (like @ghutchis) might still possess the necessary credentials to improve the situation e.g., to enable the addition of new accounts, or hint if the web site's content may be transferred into a repository on GitHub.  (Apparently, the content of openbabel.org [the web site] _is not_ within the current 0.3 GB repository of openbabel [the program].)  In case the later were true, the backlog you mention possibly could be resolved:

![main_history](https://user-images.githubusercontent.com/19709084/172887880-fad0d372-fd10-4197-8e6c-a07ff586cc74.png)

**Comment by ghutchis** [comment]
No, I simply need to find the time to clean up the draft at https://openbabel.github.io/ and replace the wiki.
